### ISSUE TITLE
Invalid prelude for @media csstree/validator

### ISSUE THREAD

**Issue by rgant** [csstree/stylelint-validator#44]
As far as I know this is valid SCSS/SASS:
```
/// Custom breakpoints
/// @access public
/// @param {String} $breakpoint - Breakpoint
/// @param {String} $from - Defaults to "min-width" (mobile first)
///        can also take "max-width" to select elements under the provided size.
@mixin breakpoint($breakpoint, $from: 'min-width') {
  @media ($from: $breakpoint) {
    @content;
  }
}
```

But it results in an error:
> src/styles/abstracts/_mixins.scss
>  25:11  ✖  Invalid prelude for `@media`  csstree/validator

My config:
.stylelintrc.yml:
```
+  csstree/validator:
+    syntaxExtensions:
+      - sass
```

I have installed: stylelint-csstree-validator@2.0.0

First referenced in csstree/csstree#118 #issuecomment-993945845

**Comment by lahmatiy** [comment]
The plugin is limited in validation of expressions which contain preprocessor's syntax (since it should be evaluated). So it avoid a validation of them. For now the plugin uses stylelint's `isStandardSyntax*()` helpers to detect non-standard syntax and some additional tricks for declaration values (but not for at-rule's preludes). Looks like `isStandardSyntaxAtRule()` helper doesn't catch such kind of expressions. So I'm going to add the same workaround for at-rule prelude as for declaration values. I think it should help.

**Comment by lahmatiy** [comment]
Fixed in [2.1.0](https://github.com/csstree/stylelint-validator/releases/tag/v2.1.0)

**Comment by pinkhominid** [comment]
I'm seeing the same error (with 2.1.0) using the postcss-lit custom syntax for expressions like this
```js
css`
    @media (min-width: 500px) and ${namedMedia.css.mobile} {
        /* ... */
    }
`
```
whereas this doesn't error
```js
css`
    @media ${namedMedia.css.mobile} {
        /* ... */
    }
`
```

**Comment by pinkhominid** [comment]
Created #52 to track this.
